src/broker/cas_execute.c

```diff
@@ -5849,14 +5849,14 @@ cgw_fetch_result (T_SRV_HANDLE * srv_handle, int cursor_pos, int fetch_count, ch
 
   if (total_row_count > INT_MAX)
     {
-      srv_handle->tuple_count = INT_MAX;
+      srv_handle->total_tuple_count = INT_MAX;
     }
   else
     {
-      srv_handle->tuple_count = (int) total_row_count;
+      srv_handle->total_tuple_count = (int) total_row_count;
     }
-  srv_handle->tuple_count = (int) total_row_count;
-  net_buf_overwrite_int (net_buf, srv_handle->total_row_count_msg_offset, srv_handle->tuple_count);
+  net_buf_overwrite_int (net_buf, srv_handle->total_row_count_msg_offset, srv_handle->total_tuple_count);	// 0
```

Comment:
remove "// 0"  and "// 10" if not necessary

Reply:
fixed it